codalab/client/json_api_client.py

```diff
@@ -559,7 +561,7 @@ def delete_relationship(
         )
 
     @wrap_exception('Unable to update authenticated user')
-    def update_authenticated_user(self, data, params=None):
+    def update_authenticated_user(self, data, params=None) -> Dict:
```

Comment:
Can we specify the types for the key and value?

Reply:
My plan is to go back to this after the general annotations are done.

Reply:
It feels inconsistent. For some methods, it looks like you type annotated the data structure elements of the return types already (e.g. https://github.com/codalab/codalab-worksheets/pull/2796/files#diff-ec08b8c99e59740ca250510b62677643R516), but for some you didn't.

Reply:
It was done in the previous stage, which was resolving errors.